packages/at_events_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.11
+- **CHORE**: Improved pub score
```

Comment:
can we change the message to "upgraded dependencies"

Reply:
@sachins-geekyants sure